src/main/java/com/craftaro/ultimateclaims/claim/Claim.java

```diff
@@ -27,13 +27,7 @@
 import org.bukkit.boss.BossBar;
 import org.bukkit.entity.Player;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
I prefer explicit imports (no `.*`) in almost all cases - Exception being automated tests for example